catalog-be/src/main/java/org/openecomp/sdc/be/externalapi/servlet/CrudExternalServlet.java

```diff
@@ -279,9 +280,11 @@ public Response createComponentExternal(
                 // Create the resource in the dataModel
                 if (responseWrapper.isEmpty()) {
                     resource = resourceBusinessLogic.createResource(resource, null, modifier, null, null);
-                    return buildCreatedResourceResponse(resource, responseWrapper);
+                    response = buildCreatedResourceResponse(resource, responseWrapper);
+                    return response;
                 } else {
-                    return buildErrorResponse(responseWrapper.getInnerElement());
+                    response = buildErrorResponse(responseWrapper.getInnerElement());
+                    return response;
```

Comment:
can be cleaner to put the return statement outside the if/else logic.

Reply:
thanks for the review!
yes indeed, moved the return statement